cluster/src/main/java/com/hartwig/pipeline/datatypes/DataType.java

```diff
@@ -21,6 +21,8 @@
     CHORD_PREDICTION,
     COBALT,
     AMBER,
+    VIRUSBREAKEND_SUMMARY,
```

Comment:
We only need to add datatypes for files which will either be consumed by another step or persisted to the DB. Do we plan to make a virusbreakend loader for the DB which uses these files?

Reply:
The summery contains information for report so could be used downstream in the future. And the VCF contains all breakpoints so could, and probably should, be loaded into patientDB at some point.